core/src/banking_stage/consumer.rs

```diff
@@ -577,7 +577,8 @@ impl Consumer {
             .iter()
             .filter_map(|transaction| {
                 transaction
-                    .compute_budget_limits(&bank.feature_set)
+                    .compute_budget_instruction_details()
+                    .sanitize_and_convert_to_compute_budget_limits()
```

Comment:
nit: if can add `_feature_set: &FeatureSet` to `sanitize_and_convert_to_compute_budget_limits()` as placeholder parameter, next PRs that's going to use feature_set won't have to touch these files again.

Reply:
Yeah I considered that but it just felt strange to modify that function when it wasn't related to the purpose of the PR